--- debian/ufoai.desktop~	2010-05-14 19:16:22.000000000 +0200
+++ debian/ufoai.desktop	2010-08-17 19:19:42.686336614 +0200
@@ -5,8 +5,8 @@
 GenericName=UFO: Alien Invasion
 Comment=Turn-based tactical combat against hostile alien forces
 Comment[de_DE]=Rundenbasiertes Taktikspiel
-Exec=/usr/games/ufo
-Icon=ufoai.xpm
+Exec=ufoai-wrapper.sh
+Icon=ufoai
 Terminal=false
 Type=Application
 Categories=Application;Game;StrategyGame;
--- debian/ufoded.desktop~	2010-05-14 19:16:22.000000000 +0200
+++ debian/ufoded.desktop	2010-08-17 19:19:37.134336759 +0200
@@ -5,8 +5,8 @@
 GenericName=UFO: AI Dedicated Server
 Comment=Turn-based tactical combat against hostile alien forces (server)
 Comment[de_DE]=Rundenbasiertes Taktikspiel (Server)
-Exec=/usr/games/ufoded
-Icon=ufoded.xpm
+Exec=ufoded
+Icon=ufoded
 Terminal=true
 Type=Application
 Categories=Application;Game;StrategyGame;
--- debian/uforadiant.desktop~	2010-05-14 19:16:22.000000000 +0200
+++ debian/uforadiant.desktop	2010-08-17 19:19:58.767336033 +0200
@@ -5,11 +5,10 @@
 GenericName=UFORadiant
 Comment=Map Editor for UFO: Alien Invasion
 Comment[de_DE]=Map Editor für UFO: Alien Invasion
-Exec=/usr/games/uforadiant
-Icon=uforadiant.xpm
+Exec=uforadiant-wrapper.sh
+Icon=uforadiant
 Terminal=false
-StartupNotify=false
 Type=Application
-Categories=Application;Game;StrategyGame;GTK;3DGraphics
+Categories=Application;Game;StrategyGame;GTK;Graphics;3DGraphics
 MimeType=application/x-ufomap
 StartupNotify=true
