--- src/ports/unix/unix_main.c~	2010-05-14 19:16:34.000000000 +0200
+++ src/ports/unix/unix_main.c	2010-09-14 09:04:55.035394517 +0200
@@ -366,6 +366,26 @@
 #ifndef HARD_LINKED_GAME
 	Com_Printf("------- Loading game.%s -------\n", SHARED_EXT);
 
+#ifdef PKGLIBDIR
+	/* first try the system library path */
+	Com_sprintf(name, sizeof(name), PKGLIBDIR"/game_"CPUSTRING"."SHARED_EXT);
+	Com_DPrintf(DEBUG_SYSTEM, "LoadLibrary: try %s\n", name);
+	game_library = dlopen(name, RTLD_LAZY);
+	if (game_library) {
+		Com_Printf("LoadLibrary (%s)\n", name);
+	} else {
+		Com_DPrintf(DEBUG_SYSTEM, "%s\n", dlerror());
+		Com_sprintf(name, sizeof(name), PKGLIBDIR"/game."SHARED_EXT);
+		Com_DPrintf(DEBUG_SYSTEM, "LoadLibrary: try %s\n", name);
+		game_library = dlopen(name, RTLD_LAZY);
+		if (game_library) {
+			Com_Printf("LoadLibrary (%s)\n", name);
+		} else {
+			Com_DPrintf(DEBUG_SYSTEM, "%s\n", dlerror());
+		}
+	}
+#endif 
+
 	/* now run through the search paths */
 	path = NULL;
 	while (!game_library) {
