diff -up ufoai-2.3.1-source/src/shared/images.c.zpng ufoai-2.3.1-source/src/shared/images.c
--- ufoai-2.3.1-source/src/shared/images.c.zpng	2010-11-29 17:46:35.000000000 +0100
+++ ufoai-2.3.1-source/src/shared/images.c	2012-05-15 23:57:55.255123139 +0200
@@ -33,6 +33,10 @@
 #include <jpeglib.h>
 #include <png.h>
 
+//#ifdef HAVE_ZLIB_H
+#include <zlib.h>
+//#endif
+
 /** image formats, tried in this order */
 static char *IMAGE_TYPES[] = { "tga", "png", "jpg", NULL };
 
