--- src/tools/radiant/radiant/environment.cpp~	2010-11-29 17:46:35.000000000 +0100
+++ src/tools/radiant/radiant/environment.cpp	2011-03-17 14:25:06.819424023 +0100
@@ -65,7 +65,7 @@
 	path.clear();
 #ifdef PKGDATADIR
 	const char *appPath = PKGDATADIR"/"RADIANT_DIRECTORY;
-	if (g_file_test(appPath, (GFileTest)G_FILE_TEST_IS_DIR) && g_path_is_absolute(path.c_str())) {
+	if (g_file_test(appPath, (GFileTest)G_FILE_TEST_IS_DIR)) {
 		app_path = PKGDATADIR"/"RADIANT_DIRECTORY;
 	} else
 #endif
